use-cases/index.markdown

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Similar to my comment on the overview page, could this be a top-level file instead (i.e. `use-cases.markdown`)? Especially as having two "use cases" directories (`use-cases/` and `_use_cases/`) could get confusing.

Reply:
Agreed, I'll fix.